README.md

```diff
@@ -269,7 +270,7 @@ When the Passwordless connection is SMS you must also add the `CountryCodeActivi
 ```
 
 
-Then in any of your Activities you need to initialize **PasswordlessLock**
+Then in any of your Activities you need to initialize **PasswordlessLock**. In case that your `auth0Scheme` value is different from `https` you should tell the library which scheme to use by calling `withScheme()` as pointed below.
```

Comment:
`If your `auth0Scheme` value is not `https` then you need to use `withScheme()` as shown below.